pkg/kubelet/cm/topologymanager/scope.go

```diff
@@ -94,11 +95,11 @@ func (s *scope) AddHintProvider(h HintProvider) {
 
 // It would be better to implement this function in topologymanager instead of scope
 // but topologymanager do not track mapping anymore
-func (s *scope) AddContainer(pod *v1.Pod, containerID string) error {
+func (s *scope) AddContainer(pod *v1.Pod, container *v1.Container, containerID string) error {
```

Comment:
Since this always returns `nil`, let's remove the `error` return value and make this function consistent with the other `AddContainer()` calls from other components.